install_gcs_connector.py

```diff
@@ -14,8 +14,8 @@
 def parse_args():
     p = argparse.ArgumentParser()
     p.add_argument("-k", "--key-file-path", help="Service account key .json")
-    p.add_argument("--gcs-requestor-pays-project", help="If specified, this google cloud project will be charged for access to "
-                   "requestor-pays buckets via spark/hadoop. See https://github.com/GoogleCloudDataproc/hadoop-connectors/blob/master/gcs/CONFIGURATION.md#cloud-storage-requester-pays-feature-configuration")
+    p.add_argument("--gcs-requester-pays-project", help="If specified, this google cloud project will be charged for access to "
```

Comment:
I'd vote for also keeping the --gcs-requester-pays-project flag for backward-compatibility

Reply:
Done in 1ec8c33172bcca2c45a08c071ef32d306d02bb14.